geode-redis/src/main/java/org/apache/geode/redis/internal/executor/set/GeodeRedisSetSynchronized.java

```diff
@@ -59,8 +60,19 @@ public long srem(Collection<ByteArrayWrapper> membersToRemove) {
       Set<ByteArrayWrapper> newValue = createSet(oldValue);
       newValue.removeAll(membersToRemove);
       removedCount.set(oldValue.size() - newValue.size());
+
+      // if(newValue.isEmpty()) {
```

Comment:
should this commented out code be removed?

Reply:
Yes.  We were experimenting with adding it into compute. Thank you for finding that!